pom.xml

```diff
@@ -3,7 +3,7 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.sps</groupId>
+  <groupId>com.google.capstone</groupId>
   <artifactId>sdktiktok</artifactId>
```

Comment:
I think the name should be changed from sdktiktok since we didn't agree on a name yet

Reply:
Yes I will change it when we decide upon the name.